glib-sys/src/lib.rs

```diff
@@ -740,10 +723,6 @@ pub const G_PRIORITY_HIGH_IDLE: c_int = 100;
 pub const G_PRIORITY_LOW: c_int = 300;
 pub const G_SEARCHPATH_SEPARATOR: c_int = 59;
 pub const G_SEARCHPATH_SEPARATOR_S: *const c_char = b";\0" as *const u8 as *const c_char;
```

Comment:
Platform specific

Reply:
IMHO is not

Reply:
It's `;` on Windows and `:` everywhere else

Reply:
Hm, really.
Strange that it have `constant name="SEARCHPATH_SEPARATOR_S" value=";" c:type="G_SEARCHPATH_SEPARATOR_S` in ubuntu GLib-2.0.gir.

Reply:
That is a bug in g-ir-scanner. It tries to parse macro definition without actually implementing a proper C preprocessing step. If macros have conditional definitions then it is quite likely to give wrong results. 

It would be nice to extend ABI checker to constants as well. I would expect to find a lot of incorrect things there.